Source/carthage/Formatting.swift

```diff
@@ -93,3 +93,36 @@ public struct ColorOptions: OptionsProtocol {
 			)
 	}
 }
+
+extension OutdatedCommand.UpdateAvailabilityAndApplicability {
+	fileprivate static let dict: [OutdatedCommand.UpdateAvailabilityAndApplicability: Color.Named.Color] = [
+		.updatesAvailableAllApplicable: .green,
+		.updatesAvailableSomeApplicable: .yellow,
+		.updatesAvailableNoneApplicable: .red,
+	]
```

Comment:
With my restructuring of the `enum` below, this could be a computed property instead.